src/coffee/containers/controls/controls-container.html

```diff
@@ -1,7 +1,20 @@
-<div class="meta-container-c">
-    <li pt-quality-selector torrents="detail.selected.torrents" selected="detail.currentQuality" class="quality-toggle"></li>
-    <li class="subtitles-dropdown"></li>
-    <li pt-device-selector selected="detail.currentDevice" id="player-option" class="device-dropdown"></li>
+<div class="containers-controls" layout="row" layout-align="center center">
```

Comment:
angular material layout system seems nice to position this kind of elements. Are you ok ?